app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,34 @@
+class ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventCatcher::Stream
+  attr_reader :ems, :stop_polling, :poll_sleep
+
+  def initialize(ems, options = {})
+    @ems = ems
+    @stop_polling = false
+    @from = DateTime.current.to_time.to_i.to_s
+    @poll_sleep = options[:poll_sleep] || 20.seconds
+  end
+
+  def start
+    @stop_polling = false
+    @from = DateTime.current.to_time.to_i.to_s
+  end
+
+  def stop
+    @stop_polling = true
+  end
+
+  def poll
+    loop do
+      @to = DateTime.current.to_time.to_i.to_s
+
+      ems.with_provider_connection(:service => 'events', :provider_region => 'us-south') do |api|
```

Comment:
We should be able to pull the `provider_region` out of the ems record in the `#connect` method down here https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/248/files#diff-9c1c1fe8a8cf1522f3f23dad6743d18f6c423658360cd877f66c9cb7ac3a1753R21